dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -762,6 +780,39 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return getStrippedWorkflowsAndServices(userDAO.findById(user.getId()));
     }
 
+    @POST
+    @Path("/workflow")
```

Comment:
What would a better path be?

Reply:
Do you mean I should change it to take in the user ID as a parameter?

Reply:
1. To be consistent with other similar endpoints, I think it should be `/{userId}/workflows`.
2. I lean towards POST because it isn't idempotent.

This is tricky; I'm not sure, and I could probably be swayed,